TypeCobol/Compiler/Sql/Scanner/SqlScanner.cs

```diff
@@ -43,6 +43,12 @@ public override Token GetTokenStartingFrom(int startIndex)
                 case '*':
                     currentIndex++;
                     return new Token(TokenType.MultiplyOperator, startIndex, currentIndex - 1, tokensLine);
+                case ',':
+                    currentIndex++;
+                    return new Token(TokenType.SQL_CommaSeparator, startIndex, currentIndex - 1, tokensLine);
+                case '.':
+                    currentIndex++;
+                    return new Token(TokenType.PeriodSeparator, startIndex, currentIndex - 1, tokensLine);
```

Comment:
For the tokens made of one char, you can use `ScanOneChar` method from Scanner. So move the method from `Scanner` to `AbstractScanner`, make it `protected` and then call it here for the three cases '*', ',' and '.'.